src/decays/decay.cpp

```diff
@@ -149,4 +149,155 @@ std::string strip_field_namespace(std::string const& s) {
    }
 }
 
+double hVV_4body(double *q2, size_t dim, void *params)
```

Comment:
We may write `(double *q2, size_t /* dim */, ...)` to ignore the `dim` parameter.

Reply:
Fixed in 790728a.